src/js/actions/layers.js

```diff
@@ -519,7 +521,7 @@ define(function (require, exports) {
             } else if (layer.kind === layer.layerKinds.TEXT) {
                 property = "boundingBox";
             } else {
-                property = "boundsNoEffects";
+                property = "boundsNoMask";
```

Comment:
we are still using it for shape tools on occasion ( i did this previously and it broke d3 because we had uninitiated bounds)